draft-wood-tls-external-psk-importer.md

```diff
@@ -124,23 +136,56 @@ used to derive concrete PSKs.
 
 # TLS 1.2 Compatibility
 
-Key importers do not affect TLS 1.2 in any way. Recall that TLS 1.2 permits computing the TLS PRF with
-any hash algorithm and PSK. Thus, a PSK may be used with the same KDF (and underlying HMAC hash algorithm) as
-TLS 1.3 with importers. However, critically, the derived PSK will not be the same since the importer
-differentiates the PSK via the identity and hash function. Thus, TLS 1.3 imported PSKs are distinct
-from those used in TLS 1.2 and avoid cross-protocol collisions.
+As specified, key importers do not affect TLS 1.2. Recall that TLS 1.2 permits computing the TLS PRF
+with any hash algorithm and PSK. Thus, an external PSK may be used with the same KDF (and underlying
+HMAC hash algorithm) as TLS 1.3 with importers. However, critically, the derived PSK will not
+be the same since the importer differentiates the PSK via the identity and hash function. Thus,
+TLS 1.3 imported PSKs are distinct from those used in TLS 1.2 and avoid cross-protocol collisions.
+
+However, as an added defense, keys imported for an endpoint that supports TLS 1.2 and 1.3 SHOULD
+derive a PSK specific to TLS 1.2 by using "tls12" as ImportedIdentity.label and SHA256 as
+ImportedIdentity.hash. The hint for this imported PSK is not modified.
 
 # Security Considerations
 
 This is a WIP draft and has not yet seen significant security analysis.
 
+# Privacy Considerations
```

Comment:
I'm a little confused what this section is meant to describe. It seems to have a sketch of something, but nothing terribly concrete yet. Were you thinking that there would ultimately be an actual mechanism in here, or is this just notes for a future draft?

Reply:
Yeah, it's just meant to be a note for an idea to fill out later, if it's something the WG should want.